test/cookbooks/php_test/recipes/phpinfo_apache.rb

```diff
@@ -1,6 +1,6 @@
 major_version = node['platform_version'].to_i < 8 ? node['php']['version'].to_i : 7
 
-::Chef::Resource.send(:include, Apache2::Cookbook::Helpers)
+include Apache2::Cookbook::Helpers
```

Comment:
This actually needs to be changed to this:
```suggestion
::Chef::Resource.include Apache2::Cookbook::Helpers
```